package.json

```diff
@@ -52,7 +52,7 @@
     "report-dir": "./coverage"
   },
   "peerDependencies": {
-    "fdk-client-javascript": "git+ssh://git@github.com/gofynd/fdk-client-javascript.git#>=0.1.15"
+    "fdk-client-javascript": "git+ssh://git@github.com/gofynd/fdk-client-javascript.git#fyndx1"
```

Comment:
No need to change this for now. We will do final update of version when SDK changes are released.